src/lossfunctions.jl

```diff
@@ -113,6 +114,63 @@ end
 Returns a function to calculate scaled Gauss loss.
 Check the help of `scaled_gauss_aux`.
 """
-function ScaledGauss()
-    return scaled_gauss_aux
+function ScaledGauss(read_var=1)
+    return (µ, meas, storage=copy(µ)) -> scaled_gauss_aux(µ, meas, storage, read_var=read_var)
+end
+
+"""
+    anscombe_aux(μ, meas, storage=copy(μ))
+
+Calculates the Poisson loss using the Anscombe-based norm for `μ` and `meas`.
+`μ` can be of larger size than `meas`. In that case
+we extract a centered region from `μ` of the same size as `meas`.
+""" 
+function anscombe_aux(μ, meas, storage=copy(μ); b=1, sqrt_meas_b=0)
+    # due to numerical errors, μ can be negative or 0
+    mm = minimum(μ)
+    if mm <= 0
+        μ .= μ .+ eps(maximum(μ)) .+ abs.(mm)
+    end
+    storage .= abs2.(sqrt_meas_b .- sqrt.(μ .+ b))
+    return sum(storage)
+    #@tullio s = @inbounds abs2(sqrt_meas_b[i] - sqrt(μ[i] + b))
+    #return s
+end
+
+
+ # define custom gradient for speed-up
+ # ChainRulesCore offers the possibility to define a backward AD rule
+ # which can be used by several different AD systems
+function ChainRulesCore.rrule(::typeof(anscombe_aux), μ, meas, storage=copy(μ); b=1, sqrt_meas_b=0)
+    Y = anscombe_aux(μ, meas, storage; b=b, sqrt_meas_b=sqrt_meas_b)
+    function anscombe_aux_pullback(xbar)
+        # @show xbar
+        # @tullio storage[i] = @inbounds (one(eltype(μ)) - sqrt_meas_b[i] / sqrt(μ[i] + b)) # xbar[i]
+        storage .= (one(eltype(μ)) .- sqrt_meas_b ./ sqrt.(μ.+b)) #xbar .* 
+        return zero(eltype(μ)), storage, zero(eltype(μ)), zero(eltype(storage)), zero(eltype(b)), zero(eltype(μ))
+    end
+
+    return Y, anscombe_aux_pullback
+end
+
+
+"""
+    Anscombe()
+
+Returns a function to calculate Poisson loss using the Anscombe transform
+Check the help of `anscombe_aux`.
+"""
+
+function Anscombe(b=1)
+    CallCounter = 0;
+    sqrt_meas_b = 0;
+    function call_brancher(μ, meas, storage=copy(µ)) # uses a closure to deal with possibly normalized measurement data.
+        if CallCounter == 0
```

Comment:
Call Counter can be removed since Optim.jl takes track of how often the functions are called (`fg!`)

Reply:
The role of the CallCounter is to ensure that the sqrt(measured + b) is calculated only once. However this is hacky and dangerous! I would prefer a different mechanism, but this means that the core of DeonvOptim needs a larger change.